code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -1,3 +1,7 @@
+#define STARLIGHT_CAN_HEAL 2
+#define STARLIGHT_CAN_HEAL_WITH_PENALTY 1
+#define STARLIGHT_CANNOT_HEAL 0
```

Comment:
might wanna bring these closer to the code for starlight condensation

Reply:
just did that thanks